server/api/db/sqldb/models.py

```diff
@@ -869,6 +869,38 @@ class TimeWindowTracker(Base, mlrun.utils.db.BaseModel):
         def get_identifier_string(self) -> str:
             return f"{self.key}"
 
+    class AlertActivation(Base, mlrun.utils.db.BaseModel):
+        __tablename__ = "alert_activation"
+        __table_args__ = (
+            PrimaryKeyConstraint("activation_time", "id", name="_alert_activation_uc"),
+            Index("ix_alert_activation_project_name", "project", "name"),
+            Index("ix_alert_activation_activation_time", "activation_time"),
+        )
+
+        id = Column(Integer)
+        activation_time = Column(SQLTypesUtil.datetime(), nullable=False)
+        name = Column(String(255, collation=SQLTypesUtil.collation()), nullable=False)
+        project = Column(
+            String(255, collation=SQLTypesUtil.collation()), nullable=False
+        )
+        data = Column(JSON)
+        entity_id = Column(
+            String(255, collation=SQLTypesUtil.collation()), nullable=False
+        )
+        entity_kind = Column(
+            String(255, collation=SQLTypesUtil.collation()), nullable=False
+        )
+        event_kind = Column(
+            String(255, collation=SQLTypesUtil.collation()), nullable=False
+        )
+        severity = Column(
+            String(255, collation=SQLTypesUtil.collation()), nullable=False
+        )
+        number_of_events = Column(Integer, nullable=False)
+
+        def get_identifier_string(self) -> str:
+            return f"{self.id}"
```

Comment:
since you have already the project name column, add the project to the identifier string. It will make it easier for debugging (it is printed in the logs in case of failures).
```suggestion
        def get_identifier_string(self) -> str:
            return f"{self.project}/{self.id}"
```

Reply:
@TomerShor but we can have many alerts per project, how it will help?

Reply:
This method is used in [here](https://github.com/mlrun/mlrun/blob/4a6e0aec96d781836d6497e130ce0647580be62d/server/api/db/sqldb/db.py#L4669-L4672) for logging only.
IMO It makes more sense that the log will show the project, and not only a random id, so we'll have more information (the project at the very least) on the record that failed to commit, instead of having to look in the DB what is this record.

Reply:
@TomerShor I think we can use {project}/{alert_name}/{id} then. What do you think?

Reply:
@rokatyy even better 👍